autonomy/cli/helpers/chain.py

```diff
@@ -159,6 +159,7 @@ def mint_component(  # pylint: disable=too-many-arguments, too-many-locals
     owner: Optional[str] = None,
     password: Optional[str] = None,
     skip_hash_check: bool = False,
+    skip_dependencies_check: bool = False,
```

Comment:
This should also be set by the framework so user doesn't have to. Rule: if l2 (network other than ethereum) don't need it ever. If Network ethereum, give user choice.